scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -414,13 +414,13 @@ def motr_config_k8(self):
         raise MotrError(errno.EINVAL, "libfabric is not up.")
 
     # Update motr-hare keys only for storage node
```

Comment:
update comment to say storage or data nodes.

Reply:
comment updated